gdk/build_system/Zip.py

```diff
@@ -38,10 +39,21 @@ def build(self, **kwargs):
             artifacts_zip_build = Path(zip_build).joinpath(utils.get_current_directory().name).resolve()
             utils.clean_dir(zip_build)
             logging.debug("Copying over component files to the '{}' folder.".format(artifacts_zip_build.name))
+            root_directory_path = utils.get_current_directory()
```

Comment:
so this is going to do what I recommended to do (which is good of course :) ). However, it would be even better if we could still exclude things from subdirectories if we want to.

So I should be able to write a glob like `test/*` and also `*/test/*`